examples/quest-1-simple/qwiklabs.yaml

```diff
@@ -0,0 +1,51 @@
+entity_type: Quest
+schema_version: 1
+
+id: my-awesome-quest
+
+tags: [sample, life-changing, gcp]
+logo: ./quest-logo.png
+default_locale: en
+
+
+# Localized meta data that's been compiled from multiple locale files.
+title:
+  locales:
+    en: Best Quest Ever
+
+description:
+  locales:
+    en: I kid you not. It's the best quest ever!
+
+objectives:
+  locales:
+    en: |
+      <p>This course will teach people</p>
+      <ul><li>The power of frienship</li></ul>
+
+audience:
+  locales:
+    en: |
+      <p>This course is intended for people who</p>
+      <ul><li>Do not understand the power of frienship</li></ul>
+
+prerequisites:
+  locales:
+    en: <p>A firm understanding of interpersonal interactions is recommended but not required.</p>
+
+suggested_duration:
+  locales:
+    en: 1 day
```

Comment:
Did you have a particular use case in mind where different locales would take different amounts of time?

Reply:
I wasn't thinking about using this input for different amounts of time per locale, but to allow the author to give a much more flexible input than N units. So we don't have to peg it to a specific unit and make the author do the conversion.

- one day
- one week
- a few hours
- 2-3 days
- A semester... seriously, pace yourself.
- A lifetime

Reply:
I definitely see the benefit to that but I'm concerned about automatically gleaning information from a pseudo-free-form text field (not that we do that now with an integer duration).